src/igvfd/types/sample.py

```diff
@@ -198,7 +198,8 @@ class Biosample(Sample):
         Path('sample_terms', include=['@id', 'term_name']),
         Path('disease_terms', include=['@id', 'term_name']),
         Path('treatments', include=['@id', 'purpose', 'treatment_type', 'summary', 'status']),
-        Path('modifications', include=['@id', 'modality', 'summary', 'status'])
+        Path('modifications', include=['@id', 'modality', 'summary', 'status']),
+        Path('institutional_certificates', include=['@id', 'certificate_identifier', 'urls'])
```

Comment:
Not sure if this was previously discussed (if so can ignore my comment) -- should we include URLs? I am just wondering if it would be useful to embed or not.

Reply:
I thought they could be eventually used for UI purposes on the biosample page, but I'll leave them out for now